db/migrate/20220816233656_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,7 @@
+# rubocop:disable all
+
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] Same here for comments, Please fix this migration to add the reference between `comments` table and `users` table as `author_id` not `user_id`.

[hint] same goes here you can add this:
```ruby
add_reference :comments, :author, null: false, foreign_key: { to_table: :users }
```